docs/scos/user/intro-to-spryker/contribute-to-documentation/build-the-documentation-site.md

```diff
@@ -28,54 +28,119 @@ We use [Jekyll](https://jekyllrb.com/) to build the Spryker documentation site.
 
 ## Prerequisites
 
-Before you begin, install the following:
+To use Jekyll, you also need to install the following:
 
 * [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.4.0 or higher, including all development headers. To check your version, run `ruby -v`.
 * [RubyGems](https://rubygems.org/pages/download), the latest version. To check your RubyGems version, run `gem -v`.
 * [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/). To check your GCC version, run `gcc -v,g++ -v`, for Make version run `make -v`.
+
+
+{% info_block warningBox "MacOS" %}
+
+If you are using MacOS, we recommend you do **not** install these prerequisites before you begin. Depending on your Mac processor and your previously installed applications, installing these before reading the instructions below may cause your installation to fail.
+
+{% endinfo_block %}
   
 ## Install Jekyll 
 
 Depending on your operating system, follow the Jekyll installation guides below.
 
 ### Install Jekyll on MacOS
 
-To install Jekyll on MacOS:
+{% info_block infoBox "Determine Your Mac's Processor Type" %}
+
+The installation process is different depending whether you are using a Mac with an Intel processor or with a Silicon M1 chip. To find out whether you have an Intel or a Silicon M1 Mac, click the **Apple** menu and choose **About This Mac**.
+
+* Silicon Macs with the M1 processor show an item labeled _Chip_ followed by the name of the Apple chip.
+
+* Intel-based Macs show an item labeled _Processor_ followed by the name and/or model number of the Intel processor.
+
+{% endinfo_block %}
+
 
-#### 1. Install command line tools
-To compile native extensions, install the command line tools:
+{% info_block warningBox "MacOS on M1 processor – Open Terminal using Rosetta" %}
 
+On a **MacBook with the M1 processor**, make sure your Terminal is opened with Rosetta 2, by following the instructions here. Rosetta is a translation layer that enables non-native Intel x86 apps, including Homebrew, to run on Apple Silicon Macs. 
+To open your terminal using Rosetta:
+1.	Open a Finder window.
+2.	In Applications, locate Terminal.
+3.	Right click on **Terminal** and select **Get Info**. 
+4.	Make sure _Open using Rosetta_ is checked. 
+  
+{% endinfo_block %}
+
+On either an M1 or an Intel Mac, follow the steps below to install Jekyll. M1-specific instructions appear as notes wherever necessary.  
```

Comment:
I converted the first info box to normal text. I agree that it's fine as plain text and that 2 consecutive notes are hard to read. I left the second box as a warning because I think it's critical information.